gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -174,8 +201,10 @@ public synchronized long getPos() throws IOException {
   @Override
   public synchronized void seek(long pos) throws IOException {
     logger.atFiner().log("seek(%d)", pos);
+    Stopwatch stopwatch = Stopwatch.createStarted();
     try {
       channel.position(pos);
+      captureDuration(SEEK_METHOD, stopwatch, 0);
```

Comment:
bytes_read do not make sense for seek, correct?

Reply:
yes, that's why passed 0 to reuse the same private method.

Reply:
It is better to add another method, rather than having a field which do not have a meaning in the case of seek()

Reply:
Done.